docs/configuration.adoc

```diff
@@ -71,6 +71,10 @@ Let's see valid attributes:
 |removeVolumes
 |Boolean to set if Cube should also remove the volumes associated with a container when removing the container. By default is true. Can be overwritten on container level.
 
+|cleanBuildImage
+|Boolean to set if Cube should also remove the images on which container is build after removing the container. By default is true. Can be overwritten on container level.
```

Comment:
No it cannot be overwritten on container level, if you set to true all built images are removed and if false no built images are removed.